src/main/java/org/sagebionetworks/web/client/widget/entity/EntityPageTop.java

```diff
@@ -370,6 +401,11 @@ public void configureDiscussionTab() {
 		}
 		discussionTab.configure(projectId, projectHeader.getName(), discussionAreaToken, canModerate, moderatorIds);
 	}
+
+	public void configureDockerTab() {
+		String projectId = projectHeader.getId();
```

Comment:
Current entity could be the repoentity, so it will need both (like the tables tab).  Something like:
`dockerTab.setProject(projectHeader.getId(), projectBundle, projectBundleLoadError);`
`dockerTab.configure(entity, entityUpdateHandler, tablesAreaToken);`